doc/docs/master/how-tos/basic-data-operations/sql-ingest.md

```diff
@@ -35,8 +35,29 @@ Where the parameters passed to the jsonnet pipeline spec are:
 | `format`      | The type of your output file containing the results of your query (either `json` or `yaml`).|
 | `secretName`  | The kubernetes secret name that contains the [password to the database](#database-secret).|
 
+!!! Example 
+
+    In this example, we are leveraging Snowflake's support for queries traversing semi-structured data (here, JSON).
+
+    - Find the [documentation for the support of semi-structured data in Snowflake](https://docs.snowflake.com/en/user-guide/semistructured-concepts.html#:~:text=Snowflake%20provides%20native%20support%20for,fast%20and%20efficient%20SQL%20querying.) here. 
+
+    - The query in the following example will use the WEATHER schema in the public test database SNOWFLAKE_SAMPLE_DATA in the COMPUTE_WH warehouse. The column V of the table DAILY_14_TOTAL stores JSON files.
+
+        Note the references to the JSON dataset elements by their hierarchical paths in the query:
+
+          ```shell
+          pachctl update pipeline --jsonnet https://raw.githubusercontent.com/pachyderm/pachyderm/master/src/templates/sql_ingest_cron.jsonnet  
+          --arg name=mysnowflakeingest 
+          --arg url="snowflake://username@GVCNUNW-MF57120/SNOWFLAKE_SAMPLE_DATA/WEATHER?warehouse=COMPUTE_WH" 
```

Comment:
Replace `GVCNUNW-MF57120`, since that's our real Snowflake account name and organization.

Reply:
my mistake - I anonymized one of the 2 urls mentioned in the page and missed the other.

Reply:
I think you are still using the real one here. Why not just use placeholder names like `<organization-account_name>`?